tests/test_parser.py

```diff
@@ -29,15 +27,28 @@ def bytes_to_service_info(payload: bytes) -> BluetoothServiceInfo:
     )
 
 
-def test_parsing_outdoor():
+def test_parsing_v5():
     device = RuuvitagBluetoothDeviceData()
-    advertisement = bytes_to_service_info(OUTDOOR_SENSOR_DATA)
+    advertisement = bytes_to_service_info(V5_SENSOR_DATA)
     assert device.supported(advertisement)
     up = device.update(advertisement)
-    expected_name = "RuuviTag EFAF"
+    expected_name = "RuuviTag DCFE"
     assert up.devices[None].name == expected_name  # Parsed from advertisement
     assert up.entity_values[KEY_TEMPERATURE].native_value == 7.2  # Celsius
     assert up.entity_values[KEY_HUMIDITY].native_value == 61.84  # %
     assert up.entity_values[KEY_PRESSURE].native_value == 1013.54  # hPa
     assert up.entity_values[KEY_VOLTAGE].native_value == 2395  # mV
     assert up.entity_values[KEY_MOVEMENT].native_value == 114  # count
+
+
+def test_parsing_v3():
+    device = RuuvitagBluetoothDeviceData()
+    advertisement = bytes_to_service_info(V3_SENSOR_DATA)
+    assert device.supported(advertisement)
+    up = device.update(advertisement)
+    expected_name = "RuuviTag DCFE"
+    assert up.devices[None].name == expected_name  # Parsed from advertisement
```

Comment:
The "Parsed from advertisement" (copy-pasted) comment can't be true here since V3 doesn't broadcast the address in the advertisement 😄